plugins/grpc-plugin/src/main.rs

```diff
@@ -31,13 +31,17 @@ async fn main() -> Result<()> {
     let plugin = Builder::new(state.clone(), tokio::io::stdin(), tokio::io::stdout())
         .option(options::ConfigOption::new(
             "grpc-port",
-            options::Value::Integer(50051),
+            options::Value::Integer(-1),
             "Which port should the grpc plugin listen for incoming connections?",
         ))
         .start()
         .await?;
 
     let bind_port = match plugin.option("grpc-port") {
+        Some(options::Value::Integer(-1)) => {
+            log::warn!("`grpc-port` option is not configured, exiting.");
```

Comment:
This is probably just log::info?  It's a great plugin, but every non-activated plugin should probably not shout to be activated :)